### PR TITLE
Add 2015-08-14 weekly update

### PR DESCRIPTION
**Pull request by diagramatics**
It is very late, unfortunately, but here it is.

### CODE REVIEW

File: locale/en/blog/weekly-updates/weekly-update.2015-08-14.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+If you have spotted or written something about Node.js and io.js, do come over to our [Evangelism team repo](https://github.com/nodejs/evangelism) and suggest it on the [Issues page](https://github.com/nodejs/evangelism/issues), specifically the Weekly Updates issue.
+
+### Upcoming Events
+
+* [BrazilJS Conf](http://braziljs.com.br/) tickets are on sale, August 21st - 22nd at Shopping Center BarraShoppingSul
+* [NodeConf EU](http://nodeconf.eu/) tickets are on sale, September 6th - 9th at Waterford, Ireland
+* [Node.js Italian Conference](http://nodejsconf.it/) tickets are on sale, October 10th at Desenzano - Brescia, Italy
+* [JSConf CO](http://www.jsconf.co/), October 16th - 17th at Ruta N, Medellin
+* [Nodevember](http://nodevember.org/?utm_source=io.js+and+Node.js+News&utm_medium=article), November 14th - 15th at Nashville, Tennessee, US.
```
Comment (fhemberger) [inline locale/en/blog/weekly-updates/weekly-update.2015-08-14.md]: Do we want to have these tracking parameters here on the blog as well?

File: locale/en/blog/weekly-updates/weekly-update.2015-08-14.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+Jump on the conversation on the [nodejs/node issue page](https://github.com/nodejs/node/issues/2327).
+
+### Community Updates
+
+* We *still* need a V8 maintainer for our LTS build! Head on over [to GitHub](https://github.com/nodejs/LTS/issues/28) to see if the requirements match your capabilities.
+* As mentioned up top, io.js v4.x will be Node.js v4.x. Or as [Mikeal Rogers](http://twitter.com/mikeal) said, [4.0 is the new 1.0](https://medium.com/node-js-javascript/4-0-is-the-new-1-0-386597a3436d).
+* The Node.js Collaborator Summit was hosted on 6th – 7th August in San Fransisco. You can see the notes in the [nodejs/summit](https://github.com/nodejs/summit) repository, [the GitHub issue](https://github.com/nodejs/summit/issues/11), or on [the Google Docs folder](https://drive.google.com/folderview?id=0B4tYrG9tjAW0fk9VOG52R2VQMmZHNWdpYnpqWU5MYUlfamNycktmeEl3UkNhdk1PS1Rid3c&usp=sharing).
```
Comment (fhemberger) [inline locale/en/blog/weekly-updates/weekly-update.2015-08-14.md]: Space is missing before the link, also on Medium.

File: locale/en/blog/weekly-updates/weekly-update.2015-08-14.md (hunk 1 above)
Comment (diagramatics) [inline locale/en/blog/weekly-updates/weekly-update.2015-08-14.md]: To be honest, Evangelism hasn't decided about using them, so I just continued putting them after the PR passes.

Comment (fhemberger) [inline locale/en/blog/weekly-updates/weekly-update.2015-08-14.md]: Okay, just askin'

File: locale/en/blog/weekly-updates/weekly-update.2015-08-14.md (hunk 2 above)
Comment (diagramatics) [inline locale/en/blog/weekly-updates/weekly-update.2015-08-14.md]: It is there. For some reason Medium doesn't have it.

**Pull request merged**